app/src/timebar/components/SpeedButton.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import iconStyles from 'styles/icons.scss';
+import buttonStyles from 'styles/components/button.scss';
+import SpeedArrowIcon from '-!babel-loader!svg-react-loader!assets/icons/arrow-speed.svg?name=SpeedArrowIcon';
+
+class SpeedButton extends Component {
+
+  changeSpeed() {
+    const ratio = this.props.decrease ? 0.5 : 2;
+    window.speed *= ratio;
```

Comment:
Window.speed was a temporary hack, maybe now is a good time to do something better. Do you think we could add the speed param to the app store, so that it could properly be sent to the Timebar container and component?

Additionally:
- speed should be limited (do it in an action or reducer), especially or low speeds where playback could appear 'broken'
- we will need to store the speed in the workspace: the typical use case where speed needed to be customized was typically with sharing a workspace

Reply:
Nice comments! I will add it to the store and workspace. I was not sure how will it work from the store at render time